libs/scala.json

```diff
@@ -1,12 +1,15 @@
 [
   {"id":"scala.akka","imports":["akka.actor.","akka.event.","akka.camel.","akka.cluster."],"name":"Akka","repo":"akka/akka/tree/master/akka-actor","tags":["actor-model"],"tech":["app-arch"],"status":"model-ready"},
   {"id":"scala.akka-streams","imports":["akka.stream."],"name":"Akka Streams","repo":"akka/akka/tree/master/akka-stream","tags":["stream-processing"],"tech":["app-arch"],"status":"model-ready"},
+  {"id":"scala.sangria","imports":["sangria."],"name":"Sangria","repo":"sangria-graphql/sangria","tags":["api-framework"],"tech":["app-arch"],"status":"awaiting-model"},
```

Comment:
It looks like it's a graphql library. let's make tags = ["graphql"]. And maybe tech  = ["storage"]?

Reply:
Will do it